tweet_stream.py

```diff
@@ -43,6 +48,12 @@ def on_data(self, data):
     def on_error(self, status):
         log.error(status)
 
+    def on_timeout(self):
+        log.error("[Tweet stream] timing out. Couldn't find tweets within 60 seconds")
+
+    def on_disconnect(self, notice):
```

Comment:
where is this used?

Reply:
We override this for the listener, in case it fails for some weird reason.
Tweetstream calls this.